consensus/safety-rules/Cargo.toml

```diff
@@ -20,6 +20,7 @@ libra-secure-push-metrics = { path = "../../secure/push-metrics", version = "0.1
 libra-secure-storage = { path = "../../secure/storage", version = "0.1.0" }
 libra-temppath = { path = "../../common/temppath", version = "0.1.0" }
 libra-types = { path = "../../types", version = "0.1.0" }
+libra-workspace-hack = { path = "../../common/workspace-hack", version = "0.1.0" }
```

Comment:
While I love what this will do for build times, it brings in features that might have been explicitly disabled to minimize dependencies. Is there another way to accomplish this via some ci specific run script?

Reply:
I'll have to look and see what the delta is.